index.html

```diff
@@ -0,0 +1,158 @@
+      </div>
+    </div>
+  </section>
+  <!-- the Program page -->
+  <section id="program">
+    <div>
+      <h3 class="program-headline">Main Program</h3>
+      <div class="underline"></div>
+    </div>
+    <div class="program-container">
+      <div class="main-program">
+        <div><img src="/img/program_icon_01.png" alt=""></div>
+        <h3 class="program-subheading">
+          Lecture
+        </h3>
+        <p class="program-info">
+          Listen to the speakers from various countries about the message of farming and food.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_02.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Exhibition
+        </h3>
+        <p class="program-info">
+          Appreciate various creations applying CC license of artists, organized from Art Center Iqbal
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_03.png" alt=""></div>
+        <h3 class="program-subheading">
+          Forum
+        </h3>
+        <p class="program-info">
+          Have the time to share your thoughts and opinions with experts for each topic.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_04.png" alt=""></div>
+        <h3 class="program-subheading">
+          Workshop
+        </h3>
+        <p class="program-info">
+          Try creating your own work using open source license rather than just watching at it.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_05.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Ignite
+        </h3>
+        <p class="program-info">
+          Get opportunities to network with HG affiliates around the world, also after the summit.
+        </p>
+      </div>
+    </div>
+    <button type="button" class="join-btn">Join HG Global Summit 2023</button>
+  </section>
+  <!-- the Speakers page -->
+  <section><h3 class="speaker-headline">Featured Speakers</h3></section>
+  <div id="speakers">    
+    
+  </div>
+  </section>
+  <!-- the partners section -->
+  <div class="partners-div"><div class="partner-title"><h3>Partners <hr class="line"></h3></div>
+    <div class="partners">
+      <img src="/img/Partners/mozilla 1.png" alt="mozilla" class="partner-logo">
+      <img src="/img/Partners/googlekorea 1.png" alt="google" class="partner-logo">
+      <img src="/img/Partners/naver 1.png" alt="naver" class="partner-logo">
+      <img src="/img/Partners/daumkakaologo 1.png" alt="daumkakaologo" class="partner-logo">
+      <img src="/img/Partners/airbnb 1.png" alt="airbnb" class="partner-logo">
+    </div>
+  </div>
+  </section>
+  <footer>
+    <div class="cc"><img src="/img/logo.jpg" alt="logo">
+      <p class="rights">2023 HopeGrower Gold Pakistan. Some Rights Reserved.</p>
```

Comment:
- [ ] please the footer section of the desktop homepage has a white background.

Reply:
I have followed your instructions and implemented the changes you suggested sir.